src/command/check-secret-expiry.ts

```diff
@@ -13,14 +13,18 @@ const getKeyAgeStatus = (
   const daysLeft = keyMaxAge - keyAge;
 
   if (daysLeft < 0) {
-    return { text: `✘ maximum age is exceeded`, textColor: chalk.red };
+    return { text: `✘ maximum age has exceeded`, textColor: chalk.red };
```

Comment:
dude. 
I told you this should say `has been exceeded` 
I explicitly left that as a comment.

Reply:
sorry I actually said "will be reached" ... either way -- you just resolved the comment, made no changes, and then re-requested me.